<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_file</class>
 <widget class="QWidget" name="add_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Widget{border-image: url(:/new/prefix1/background.png);}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-color: rgb(255, 255, 127);}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_txt">
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(170, 255, 255);</string>
       </property>
       <property name="text">
        <string>添加文本</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_music">
       <property name="text">
        <string>添加音频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_pic">
       <property name="text">
        <string>添加图片</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>add_music</zorder>
    <zorder>add_pic</zorder>
    <zorder>add_txt</zorder>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>161</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_music">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>161</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_picture">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>161</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(0, 255, 121, 255), stop:0.744318 rgba(0, 255, 196, 255), stop:1 rgba(0, 255, 229, 255));</string>
    </property>
    <property name="text">
     <string>选择素材保存路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>add_file</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
